src/chains-config/calamariControllers.ts

```diff
@@ -0,0 +1,37 @@
+import { ControllerConfig } from '../types/chains-config';
+import { initLRUCache } from './cache/lruCache';
+import { getBlockWeight } from './metadata-consts';
+
+/**
+ * Controllers for calamari collator
+ */
+export const calamariControllers: ControllerConfig = {
+	controllers: [
+		'AccountsBalanceInfo',
+		'AccountsStakingInfo',
+		'AccountsStakingPayouts',
+		'AccountsValidate',
+		'AccountsVestingInfo',
+		'Blocks',
+		'BlocksExtrinsics',
+		'NodeNetwork',
+		'NodeTransactionPool',
+		'NodeVersion',
+		'PalletsStakingProgress',
+		'PalletsStorage',
+		'Paras',
+		'RuntimeCode',
+		'RuntimeMetadata',
+		'RuntimeSpec',
+		'TransactionDryRun',
+		'TransactionFeeEstimate',
+		'TransactionMaterial',
+		'TransactionSubmit',
+	],
+	options: {
+		finalizes: true,
+		minCalcFeeRuntime: null,
```

Comment:
It's this line here that is causing it to error, set it to 1 assuming your minimum runtime version that supports fees is 1.

Reply:
thank you! that makes good sense.

Reply:
with that changed the error message now includes the runtime version. eg: "Fee calculation not supported for 4#calamari" from [/blocks/311056](https://api.serrano.calamari.systems/blocks/311056)

Reply:
Hmm, I was able to get it working locally. To test it locally did you link the calc package?

To do that inside of the package.json you will replace the version of `@substrate/calc` to -> `"file:./calc/pkg"` in order to get the updated wasm generated code.

Reply:
thank you again! i really appreciate your help with making this work. that does indeed fix it.